V3-Upgrade-Guide.md

```diff
@@ -37,3 +37,23 @@ For historic reasons the command-line arguments of both applications were differ
 In version 3, the arguments are aligned and deprecated arguments have been removed.
 Please see (insert link here) for the current documentation on the cli arguments.
 -->
+
+### Timeout 
```

Comment:
when adding new sections to the upgrade guide, please also add them to the table of contents in line 20 in this file.